packages/v4/patternfly-docs/content/design-guidelines/components/label/label.md

```diff
@@ -3,8 +3,6 @@ id: Label
 section: components
 ---
 
-The **label** component allows users to add specific element captions for user clairty and convenience
```

Comment:
Just noticed there's a typo in this one - should say "clarity" instead of "clairty"